docs/analyzer/user_guide.md

```diff
@@ -638,7 +638,7 @@ instead of the auto-detection you can pass that to the
 
 There are some implicit include paths (for example those which contain
 `include-fixed` directory) which are used only by GCC. By default CodeChecker
-doesn't collect them if `--skip-gcc-fix-include` flag is given. For
+doesn't collect them unless `--keep-gcc-include-fixed` flag is given. For
```

Comment:
What do you think about rephrasing these sentences a bit? I think it can be misunderstood, that the implicit include files will be only collected unless the  `--keep-gcc-include-fixed` flag is used. We should highlight a bit more that the include paths will be collected only the `include-fixed` directories will be skipped/removed from the list of the include paths.